client/app/components/dynamic-parameters/DynamicParameters.less

```diff
@@ -0,0 +1,36 @@
+@import '../../assets/less/inc/variables';
+
+.redash-datepicker {
+  width: auto !important;
+
+  .ant-calendar-picker-clear {
+    right: 35px;
+  }
+
+  &.date-range-input .ant-calendar-picker-input {
+    width: 376px;
```

Comment:
Too wide for other date range types I think.
Perhaps set the width at the component level?
```jsx
const widthByType = {
  'date-range': 250,
  'datetime-range': 320,
  'datetime-range-with-seconds': 370,
};

return <DateTimeRangeInput
  width={widthByType(this.props.type)}
}
```

Reply:
I went with 270 for date-range, 250 I felt the clear button was too close.